code/datums/ammo/xeno.dm

```diff
@@ -37,60 +37,66 @@
 
 	neuro_callback = CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(apply_neuro))
 
-/proc/apply_neuro(mob/living/M, power, insta_neuro)
-	if(skillcheck(M, SKILL_ENDURANCE, SKILL_ENDURANCE_MAX) && !insta_neuro)
-		M.visible_message(SPAN_DANGER("[M] withstands the neurotoxin!"))
+/proc/apply_neuro(mob/living/target_mob, power, insta_neuro)
+	if(skillcheck(target_mob, SKILL_ENDURANCE, SKILL_ENDURANCE_MAX) && !insta_neuro)
+		target_mob.visible_message(SPAN_DANGER("[target_mob] withstands the neurotoxin!"))
 		return //endurance 5 makes you immune to weak neurotoxin
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(H.chem_effect_flags & CHEM_EFFECT_RESIST_NEURO || H.species.flags & NO_NEURO)
-			H.visible_message(SPAN_DANGER("[M] shrugs off the neurotoxin!"))
+	if(ishuman(target_mob))
+		var/mob/living/carbon/human/target_human = target_mob
+		if(target_human.chem_effect_flags & CHEM_EFFECT_RESIST_NEURO || target_human.species.flags & NO_NEURO)
+			target_human.visible_message(SPAN_DANGER("[target_human] shrugs off the neurotoxin!"))
 			return //species like zombies or synths are immune to neurotoxin
 
-	if(!isxeno(M))
+	if(!isxeno(target_mob))
 		if(insta_neuro)
-			if(M.knocked_down < 3)
-				M.adjust_effect(1 * power, WEAKEN)
-			return
-
-		if(ishuman(M))
-			M.apply_effect(2.5, SUPERSLOW)
-			M.visible_message(SPAN_DANGER("[M]'s movements are slowed."))
+			if(target_mob.knocked_down < 3)
+				target_mob.adjust_effect(1 * power, WEAKEN)
+				return
 
+	if(ishuman(target_mob))
+		var/mob/living/carbon/human/target_human = target_mob
 		var/no_clothes_neuro = FALSE
-
-		if(ishuman(M))
-			var/mob/living/carbon/human/H = M
-			if(!H.wear_suit || H.wear_suit.slowdown == 0)
-				no_clothes_neuro = TRUE
-
-		if(no_clothes_neuro)
-			if(M.knocked_down < 5)
-				M.adjust_effect(1 * power, WEAKEN) // KD them a bit more
-				M.visible_message(SPAN_DANGER("[M] falls prone."))
-
-/proc/apply_scatter_neuro(mob/living/M)
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(skillcheck(M, SKILL_ENDURANCE, SKILL_ENDURANCE_MAX))
-			M.visible_message(SPAN_DANGER("[M] withstands the neurotoxin!"))
+		if(!target_human.wear_suit || target_human.wear_suit.slowdown == 0)
+			no_clothes_neuro = TRUE
+		if((target_human.dazed && target_human.superslowed) || target_human.knocked_down || no_clothes_neuro)
+			target_human.apply_effect(power, WEAKEN)
+			target_human.visible_message(SPAN_DANGER("[target_human] collapses into the ground."))
```

Comment:
Collapses to the ground
Also this message doesn't make sense if the target is already prone